.ci/validation.sh

```diff
@@ -61,7 +74,7 @@ check-missing-pitests)
 
   echo "List of missing files in pitest profiles: $results"
 
-  if [[ -n $results ]] ; then
+  if [[ -n $results ]]; then
```

Comment:
please move ALL unrelated formatting to separate commit.

what tool do you use for this ?

Reply:
None. Just Ctrl+X/Ctrl+V in IntellyJ IDEA.

Reply:
I recently update IDEA to 2020.3, this may be related.